rdkit/VLib/Filter.py

```diff
@@ -3,10 +3,10 @@
 #  Copyright (C) 2003 Rational Discovery LLC
 #     All Rights Reserved
 #
-import sys
+
+import six
```

Comment:
I find it a bit strange to have an internal copy of six (and even modify it). However, I don't feel to strongly about this and have reverted these changes.